app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name, health: int = 100):
+        self.name = name
+        self.health = health
+        self.hidden = False
+        if self.health > 0:
+            self.__class__.alive.append(self)
+
+    def __repr__(self):
+        return f"{{Name: {self.name}," \
+               f" Health: {self.health}," \
+               f" Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+    def hide(self):
+        self.hidden = False if self.hidden is True else True
```

Comment:
you can write this better
Try to write like this:
`self.hidden = not self.hidden`